index.html

```diff
@@ -3,9 +3,12 @@
 <head>
```

Comment:
Can we make the indentation two tabs? You can probably set this in your editor. Please change this for the whole HTML file.